src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensRangeHandler.cs

```diff
@@ -155,10 +155,11 @@ private void EnqueueSemanticTokenRefreshNotification(Uri? documentUri)
             Contract.ThrowIfNull(request.TextDocument, "TextDocument is null.");
             Contract.ThrowIfNull(context.Document, "Document is null.");
 
-            // If the full compilation is not yet available, we'll try getting a partial one. It may contain inaccurate
-            // results but will speed up how quickly we can respond to the client's request.
-            var document = context.Document.WithFrozenPartialSemantics(cancellationToken);
+            var document = context.Document;
             var project = document.Project;
+
+            // Currently this does nothing as OOP does not support frozen partial.  However we set this flag as an indicator that we do
+            // want to support it here eventually (tracked by https://github.com/dotnet/roslyn/issues/63968).
```

Comment:
I waffled between removing all the code handling frozen partial and just leaving it.  For now I left it since we do want to support it eventually.